pages/[id]/minigames/obstacle-jumper/index.js

```diff
@@ -0,0 +1,183 @@
+import kaboom from "kaboom";
+import { useRef, useEffect } from "react";
+import { useRouter } from "next/router";
+
+export default function ObstacleJumper({ onAddMoney, myPets }) {
+  const router = useRouter();
+  const id = router.query.id;
+  const canvasRef = useRef(null);
+
+  useEffect(() => {
+    const k = kaboom({
+      fullscreen: true,
+      scale: 1,
+      global: false,
+      background: [214, 210, 210],
+      canvas: canvasRef.current,
+    });
+
+    const actualPet = myPets.find((myPet) => myPet.id === id);
+
+    if (!actualPet) {
+      return;
+    }
+
+    let score = 0;
+
+    //Scene Management
+
+    k.scene("start", () => {
+      const start = k.add([
+        k.text("Press to Start"),
```

Comment:
It wasn't clear to me how to start the game.

```suggestion
        k.text("Press Space to Start"),
```

Reply:
fixed !